webpack.config.js

```diff
@@ -60,12 +60,18 @@ function buildConfig(envConfig) {
       "./source-editor": "devtools/client/sourceeditor/editor",
       "./test-flag": "devtools/shared/flags",
       react: "devtools/client/shared/vendor/react",
+      redux: "devtools/client/shared/vendor/redux",
       "react-dom": "devtools/client/shared/vendor/react-dom",
       lodash: "devtools/client/shared/vendor/lodash",
+      immutable: "devtools/client/shared/vendor/immutable",
+      "react-redux": "devtools/client/shared/vendor/react-redux",
+
       "wasmparser/dist/WasmParser": "devtools/client/shared/vendor/WasmParser",
       "wasmparser/dist/WasmDis": "devtools/client/shared/vendor/WasmDis",
+      "../assets/panel/debugger.properties": "devtools/shared/flags",
```

Comment:
If I understand correctly this redirect works because those dependencies are not really used when isFirefoxPanel is true, and flags is just used randomly because it is one of our externalized dependencies? Could we add a comment here to explain the trick?

Reply:
By the way, I'm just thinking that we could both clarify the intent by importing a dedicated external module that throws a clear error if it is required. Something like "forbidden-panel-import.js". It would make it easier to investigate regressions where a require to an excluded lib made its way back to the panel version.

Reply:
That's a nice idea. Lets land this and if we regress we can do that fix as a follow up.